compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -969,8 +970,11 @@ convertImplements :: Env -> LF.TypeConName -> ConvertM (NM.NameMap TemplateImple
 convertImplements env tpl = NM.fromList <$>
   mapM convertInterface (MS.findWithDefault [] tpl (envImplements env))
   where
-    convertInterface iface = do
-      con <- convertInterfaceTyCon env iface
+    convertInterface :: (Maybe LF.SourceLoc, GHC.TyCon) -> ConvertM TemplateImplements
+    convertInterface (originLoc, iface) = withRange originLoc $ do
+      let handleIsNotInterface tyCon =
+            conversionError $ "cannot implement '" ++ GHC.showSDocUnsafe (ppr tyCon) ++ "' because it is not an interface"
```

Comment:
```suggestion
            conversionError $ "cannot implement '" ++ prettyPrint tyCon ++ "' because it is not an interface"
```

This one is a bit hard to find, it's defined in our fork of `ghcide`, but we're already using it in this module. It's pretty much the same underneath though

Reply:
Done